interfaces/acados_matlab_octave/check_acados_requirements.m

```diff
@@ -51,27 +53,46 @@ function check_acados_requirements(varargin)
         end
 
         if ismac
-            if ~verLessThan('matlab', '8.5')
-                filename = strcat('casadi-osx-matlabR2015a-v', CasADi_version, '.tar.gz');
-            elseif ~verLessThan('matlab', '8.4')
-                filename = strcat('casadi-osx-matlabR2014b-v', CasADi_version, '.tar.gz');
-            elseif ~verLessThan('matlab', '8.3')
-                filename = strcat('casadi-osx-matlabR2014a-v', CasADi_version, '.tar.gz');
+        CasADi_version = '3.5.5'; % TODO: update automatic download to later CasADi version.
```

Comment:
It seems to be not implemented for Mac yet?
Here `CasADi_version` and `CasADi_version_numeric` are not consistent.